### ISSUE TITLE
Site is hard to use without the floating mini TOC on each page

### ISSUE THREAD

**Issue by lonix1** [MicrosoftDocs/feedback#3954]
**Describe the bug**
Documents used to show a mini floating TOC on the right hand side of each page. It's been replaced with an "Additional resources" section, which is useless and should be shown at the bottom.

Articles are much harder to grok without that mini TOC. I have to continually scroll up and down to determine where I am, or scroll to the top to look at the full TOC.

Useability of the docs site is much poorer because of this.

**To Reproduce**
Visit any page.

**Expected behavior**
Floating mini TOC on the right of every page.

**Screenshots**
An example screenshot I found of the old layout:

<img src="https://user-images.githubusercontent.com/1622880/160897360-16094c2c-822a-40d4-a75c-8d1c2be6b1ee.PNG" />

**Desktop (please complete the following information):**
all desktops

**Smartphone (please complete the following information):**
n/a

**Additional context**
-

**Comment by gewarren** [comment]
FYI @keziamicro

**Comment by gewarren** [comment]
Related to #3925.

**Comment by gewarren** [comment]
@lonix1 Yes, the time to read was removed in April 2023. This was the explanation for that:

> The feature works great for simple articles but is misleading when a customer is trying to work through more complex documentation. In our world, time to read is one thing but time to comprehend and DO is quite another. Because the “time to read” feature is hurting more than helping our users and since comprehension time is not a straightforward calculation, we’ve decided to retire it effective April 2023.
